docs/markdown/Reference-tables.md

```diff
@@ -53,6 +53,7 @@ These are return values of the `get_linker_id` (Linker family) method in a compi
 | MESON_DIST_ROOT     | Points to the root of the staging directory, only set when running `dist` scripts |
 | MESON_SOURCE_ROOT   | Absolute path to the source dir |
 | MESON_SUBDIR        | Current subdirectory, only set for `run_command` |
+| NINJA               | Absolute path to ninja executable |
```

Comment:
I don't think this is the correct place in the documentation for this.   This is a table called "Script environment variables", which (I think) lists enviroment variables set _by meson_  in the environment for scripts it runs.

There should be documentation of environment variables which affect the behaviour of meson, but not here.

Reply:
I think there could be a new section in this Reference-tablets.md file like

## Environment variables to Meson

These variables tell Meson the name or path of desired executables.
In general it's more robust to use native-file or cross-file for this purpose.

variable | description
---------|-------------
CC         | C compiler name or path. Example: `gcc` or `/usr/bin/gcc`
CXX      | C++ compiler name or path
FC         | Fortran compiler name or path 
NINJA   | path to ninja-build executable. Example:  `c:/bin/ninja.exe` or `/home/user/.local/bin/ninja`